### ISSUE TITLE
SpladeTable selectFilter() method not working correctly when array has value zero

### ISSUE THREAD

**Issue by sdiama** [protonemedia/laravel-splade#109]
Let's assume you want to filter a column 'Enabled' which id DB has values 0 or 1. So at configure() method of the table you have to write:
```
->selectFilter('enabled', [
    '1' => 'Yes',
    '0' => 'No'
]
```

When the user selects 'Yes' everything works fine. Spade created the query:
`select * from 'mytable' where (LOWER('mytable'.'enabled') = '1') limit 15 offset 0`

...but when the 'No' is selected the table does not work. In this case the created query is:
`select * from 'mytable' limit 15 offset 0`
instead of the expected:
`select * from 'mytable' where (LOWER('mytable'.'enabled') = '0') limit 15 offset 0`

**Comment by pascalbaljet** [comment]
Thanks for sharing, I'm working on a fix :)

**Comment by pascalbaljet** [comment]
Fixed in v0.7.3.
